tests/e2e/esm/index.test.ts

```diff
@@ -680,18 +718,20 @@ describe('e2e esm tests', async () => {
       }
     });
 
-    // TODO: Make a vault within this test environment's context to cover this
-    // it('should be able to get a specific vault', async () => {
-    //   const vault = await ario.getVault({
-    //     address: '31LPFYoow2G7j-eSSsrIh8OlNaARZ84-80J-8ba68d8',
-    //     vaultId: 'Dmsrp1YIYUY5hA13euO-pAGbT1QPazfj1bKD9EpiZeo',
-    //   });
-    //   assert.deepEqual(vault, {
-    //     balance: 1,
-    //     startTimestamp: 1729962428678,
-    //     endTimestamp: 1731172028678,
-    //   });
-    // });
+    it('should be able to get a specific vault', async () => {
+      const { items: vaults } = await ario.getVaults();
+      if (vaults.length > 0) {
+        const vault = await ario.getVault({
+          address: vaults[0].address,
+          vaultId: vaults[0].vaultId,
+        });
+        assert.deepEqual(vault, {
+          balance: 1,
+          startTimestamp: 1729962428678,
+          endTimestamp: 1731172028678,
+        });
+      }
```

Comment:
this doesn't seem stable. that vault will end at 1731172028678. maybe just assert the types?

Reply:
yes - good catch

Reply:
hmm, `1731172028678` is actually in `GMT: Saturday, November 9, 2024 5:07:08.678 PM`

I'm guessing this is just getting no vaults atm